src/convert.jl

```diff
@@ -194,12 +194,12 @@ function jlvec(::Type{T}, rv::RVEC, force_missing::Bool=true) where T
     end
 end
 
-function sexp2julia(rex::RSEXPREC)
+function sexp2julia(rex::RSEXPREC; metadata::Bool=true)
```

Comment:
I have no strong preference, but wouldn't
```suggestion
function sexp2julia(rex::RSEXPREC; kwargs...)
```
be more future-proof? Also it would be an indication that a given `sexp2julia()` overload doesn't implement any metadata etc processing logic.

Reply:
On the contrary, I don't like accepting (and ignoring) any passed argument. That can lead to silent bugs, and it even means that adding new keyword arguments would be breaking as people could start relying on the fact that they can pass arbitrary keyword arguments without affecting the behavior.